data-serving/data-service/api/openapi.yaml

```diff
@@ -298,25 +262,124 @@ components:
                 type: string
         travelHistory:
           type: object
+          properties:
+            traveledPrior30Days:
+              type: boolean
+            travel:
+              type: array
+              items:
+                type: object
+                properties:
+                  dateRange:
+                    $ref: "#/components/schemas/DateRange"
+                  location:
+                    $ref: "#/components/schemas/Location"
+                  purpose:
+                    type: string
+                  methods:
+                    type: array
+                    items:
+                      type: string
         genomeSequences:
           type: array
           items:
             type: object
+            properties:
+              sampleCollectionDate:
+                $ref: "#/components/schemas/Date"
+              repositoryUrl:
+                type: string
+              sequenceId:
+                type: string
+              sequenceName:
+                type: string
+              sequenceLength:
+                type: integer
+                format: int64
+                minimum: 0
         pathogens:
           type: array
           items:
             type: object
+            properties:
+              name:
+                type: string
+              id:
+                type: integer
+                format: int32
+                minimum: 0
         notes:
           type: string
         revisionMetadata:
           type: object
+          properties:
+            revisionNumber:
+              type: integer
+              format: int32
+              minimum: 0
+            creationMetadata:
+              $ref: "#/components/schemas/EditMetadata"
+            editMetadata:
+              $ref: "#/components/schemas/EditMetadata"
     CaseArray:
       type: object
       properties:
         cases:
           type: array
           items:
             $ref: "#/components/schemas/Case"
+    Date:
+      oneOf:
+        - type: string
+        - type: object
+        - type: number
+    DateRange:
+      type: object
+      properties:
+        start:
+          $ref: "#/components/schemas/Date"
+        end:
+          $ref: "#/components/schemas/Date"
+    EditMetadata:
+      type: object
+      properties:
+        curator:
+          type: string
+        date:
+          $ref: "#/components/schemas/Date"
+        notes:
+          type: string
+    Location:
+      type: object
+      properties:
```

Comment:
Should more properties (here and elsewhere) be labeled as required? E.g., location.country, metadata.curator.

Reply:
believe it or not, some of the imported data doesn't have even that. so those fields are required in NewCase, but not in Case (except metadata.curator -- trying to work through that now...)